spec.html

```diff
@@ -34289,7 +34266,7 @@ <h1>Runtime Semantics: SortCompare ( _x_, _y_ )</h1>
             <p>Because non-existent property values always compare greater than *undefined* property values, and *undefined* always compares greater than any other value, *undefined* property values always sort to the end of the result, followed by non-existent property values.</p>
           </emu-note>
           <emu-note>
-            <p>Method calls performed by the ToString abstract operations in steps 5 and 7 have the potential to cause SortCompare to not behave as a consistent comparison function.</p>
+            <p>Method calls performed by the ToString abstract operations in steps 5 and 6 have the potential to cause SortCompare to not behave as a consistent comparison function.</p>
```

Comment:
(step numbers here too)

Reply:
This was addressed upstream and resolved during rebase.